CHANGELOG.md

```diff
@@ -5,10 +5,12 @@
 - Self-hosted etcd: if `etcd-hosts.checkpoint` file exists under `${datadir}/`,
   etcd pod will restore the hosts mapping from it before etcd bootstraps.
 - Add static TLS support for self-hosted etcd mode.
+- Added an RBAC rule for the resource `events` to allow the operator to create events.
```

Comment:
That's right.
Tbh we probably should check the requirements before the pod becomes ready